src/modules/sessions-management-pane/sessions-management-pane-api/abort-all-sessions.ts

```diff
@@ -24,7 +24,7 @@ export const abortAllSession = async (
     await axios.post(`/${agentType === "ServiceGroup" ? "groups" : "agents"}/${agentId}/plugins/${PLUGIN_ID}/dispatch-action`, {
       type: "CANCEL_ALL",
     });
-    sendAlertEvent({ type: "SUCCESS", title: `(${sessionCount}) Sessions have been aborted successfully.` });
+    sendAlertEvent({ type: "SUCCESS", title: `${sessionCount > 1 ? `(${sessionCount})` : ""} Sessions have been aborted successfully.` });
```

Comment:
checking that the number of sessions is more than 1 is not needed. You can write sessionCount ? smth : else